CONTRIBUTING.md

```diff
@@ -184,6 +184,8 @@ which combinations are tested regularly.
    documentation.
  * `BUILD_TYPE`: if set, override the `CMAKE_BUILD_TYPE` flag when configuring
    the build.
+ * `BUILD_TESTING`: if set to `no` disable the unit tests, this is sometimes
```

Comment:
Comma after `no`, please.

Start a new sentence with "This is sometimes".

Reply:
Done.